frontend/src/MovieGallery/component/TopRatedMovieGallery.tsx

```diff
@@ -0,0 +1,22 @@
+import {TopRatedMovie} from "../model/TopRatedMovie";
+import TopRatedMovieComponent from "./TopRatedMovieComponent";
+
+type TopRatedMovieGalleryProps = {
+    topRatedMovies: TopRatedMovie[]
+}
+
+export default function TopRatedMovieGallery(props: TopRatedMovieGalleryProps){
+
+    const topRatedMovieComponent = props.topRatedMovies.filter((movie) => movie.vote_average >= 8)
```

Comment:
Why filter for 8? Try to avoid 'magic numbers'

Reply:
because we only want movies with a average of 8 or greater